docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: '3.4'
```

Comment:
👍 Thanks.

This started just recently bugging with [warnings](https://github.com/compose-spec/compose-spec/commit/6d19079ef64dd94d203076e9de6caeb20c498efe), TL;DR: compose 2+ doesn't use it, compose<2 needs it but can't run bedrock due to other things in services definition anyways, so this has no real world impact nowadays.